.github/workflows/ci.yml

```diff
@@ -11,22 +11,23 @@ on:
         description: 'To run with tmate enter "debug_enabled"'
         required: false
         default: "false"
-
+env:
+  GO_VERSION: 1.18.4
 jobs:
   lint:
     name: Lint
     runs-on: ubuntu-latest
     env:
       GO111MODULE: "on"
```

Comment:
We can remove GO111MODULE everywhere now, right?

Reply:
Right!